.github/release-drafter.yml

```diff
@@ -27,6 +27,19 @@ exclude-contributors:
 category-template: '### $TITLE'
 change-template: '* $TITLE ([#$NUMBER]($URL))'
 sort-by: 'title'
+replacers:
+  - search: '/@(\w+)(,\s| and )?/g'
+    replace: '- [@$1](https://github.com/$1)\\n'
```

Comment:
This is probably more for my understanding: Why are there now two backslashes (`\\`) before the `n`? In comment https://github.com/GenericMappingTools/pygmt/issues/2775#issuecomment-1784454161 it was only one backslash (`\`).

Reply:
I first tried `\n`, then found it didn't work, then tried `\\n`, still no luck.